commands/command_migrate_info.go

```diff
@@ -0,0 +1,161 @@
+package commands
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/git-lfs/git-lfs/errors"
+	"github.com/git-lfs/git-lfs/git/odb"
+	"github.com/git-lfs/git-lfs/tools"
+	"github.com/git-lfs/git-lfs/tools/humanize"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// migrateInfoTopN is a flag given to the git-lfs-migrate(1) subcommand
+	// 'info' which specifies how many info entries to show by default.
+	migrateInfoTopN int
+
+	// migrateInfoAboveFmt is a flag given to the git-lfs-migrate(1)
+	// subcommand 'info' specifying a human-readable string threshold of
+	// filesize before entries are counted.
+	migrateInfoAboveFmt string
+	// migrateInfoAbove is the number of bytes parsed from the above
+	// migrateInfoAboveFmt flag.
+	migrateInfoAbove uint64
```

Comment:
> I think this output looks great, but the file sizes should be aligned too:

Good call: I built up a `size` array as well, and aligned it in 7d2dbd6. I think it looks much better this way!